README.md

```diff
@@ -31,9 +31,18 @@ You can read this example bot on GitHub: [giveaways-bot](https://github.com/Andr
 
 ### Launch of the module
 
+Required Discord Intents: `GUILDS`, `GUILD_MESSAGES`, `GUILD_MESSAGE_REACTIONS`.  
```

Comment:
Should the first 2 stay there?
Cause technically they would not be needed _I think_
But without them you also won't get to issue a command in your bot, in the first place.
But is that relevant for us?

lmk

Reply:
its probably going to be required on like april

Reply:
Yeah, but I meant it like the first 2 aren't needed to start a gaw.
Which now proved to be wrong cause at least "GUILDS" is needed to get a channel from the clients cache.
So I will just leave it like that I guess.